test/extended/include.go

```diff
@@ -24,7 +24,6 @@ import (
 
 	// _ "k8s.io/kubernetes/test/e2e/ui"
 
-	_ "github.com/openshift/origin/test/extended/authorization"
```

Comment:
When you add this back, you do not want my helper since the live calls remove all race conditions.

Reply:
done